sklearn/utils/validation.py

```diff
@@ -26,7 +27,14 @@ class DataConversionWarning(UserWarning):
 
 class NonBLASDotWarning(UserWarning):
     "A warning on implicit dispatch to numpy.dot"
-    pass
+
+
+class NotFittedError(ValueError, AttributeError):
+    """
+    Exception class to raise if estimator is used before fitting
+
+    Inherits from both ValueError and AttributeError to help exception handling
```

Comment:
Actually the PEP257 format for docstrings is: oneliner on the first line, blank line, paragraph with details:

```python
class NotFittedError(ValueError, AttributeError):
    """Exception class to raise if estimator is used before fitting

    This class inherits from both ValueError and AttributeError to help with exception
    handling and backward compatibility.
    """

```